{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"collect_submission.ipynb","provenance":[]},"kernelspec":{"name":"python3","display_name":"Python 3"}},"cells":[{"cell_type":"code","metadata":{"id":"n12TcHrJsEhY","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1595458277358,"user_tz":-540,"elapsed":1865,"user":{"displayName":"성주용","photoUrl":"","userId":"13100374367683138825"}},"outputId":"fb2e97f2-1056-4a5d-f7af-4ffbb2153947"},"source":["from google.colab import drive\n","\n","drive.mount('/content/drive', force_remount=True)\n","\n","# enter the path in your Drive to assignment1\n","# e.g. 'cs231n/assignments/assignment1/'\n","FOLDERNAME = 'cs231n/assignments/assignment1/'\n","\n","assert FOLDERNAME is not None, \"[!] Enter the foldername.\""],"execution_count":6,"outputs":[{"output_type":"stream","text":["Mounted at /content/drive\n"],"name":"stdout"}]},{"cell_type":"markdown","metadata":{"id":"B9ITM0nQsEhd","colab_type":"text"},"source":["# Collect Submission - Zip + Generate PDF \n","\n","Run this notebook once you have completed all the other notebooks: `knn.ipynb`, `svm.ipynb`, `softmax.ipynb`, `two_layer_net.ipynb` and `features.ipynb`).\n","\n","It will:\n","\n","* Generate a zip file of your code (`.py` and `.ipynb`) called `a1.zip`.\n","* Convert all notebooks into a single PDF file called `assignment.pdf`.\n","\n","If your submission for this step was successful, you should see the following display message:\n","\n","`### Done! Please submit a1.zip and the pdfs to Gradescope. ###`\n","\n","Make sure to download the zip and pdf file locally to your computer, then submit to Gradescope. Congrats on succesfully completing the assignment!"]},{"cell_type":"code","metadata":{"id":"p0rHoMZbsEhd","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":895},"executionInfo":{"status":"error","timestamp":1595458280327,"user_tz":-540,"elapsed":946,"user":{"displayName":"성주용","photoUrl":"","userId":"13100374367683138825"}},"outputId":"9eb2e073-001a-4cc2-e007-fed626dd9168"},"source":["%cd drive/My\\ Drive\n","%cd $FOLDERNAME\n","!sudo apt-get install texlive-xetex texlive-fonts-recommended texlive-generic-recommended\n","!pip install PyPDF2\n","!bash collectSubmission.sh"],"execution_count":7,"outputs":[{"output_type":"stream","text":["ERROR:root:Internal Python error in the inspect module.\n","Below is the traceback from this internal error.\n","\n"],"name":"stderr"},{"output_type":"stream","text":["Traceback (most recent call last):\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/interactiveshell.py\", line 2882, in run_code\n","    exec(code_obj, self.user_global_ns, self.user_ns)\n","  File \"<ipython-input-7-af39bf57339a>\", line 1, in <module>\n","    get_ipython().magic('cd drive/My\\\\ Drive')\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/interactiveshell.py\", line 2160, in magic\n","    return self.run_line_magic(magic_name, magic_arg_s)\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/interactiveshell.py\", line 2081, in run_line_magic\n","    result = fn(*args,**kwargs)\n","  File \"<decorator-gen-91>\", line 2, in cd\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/magic.py\", line 188, in <lambda>\n","    call = lambda f, *a, **k: f(*a, **k)\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/magics/osm.py\", line 288, in cd\n","    oldcwd = py3compat.getcwd()\n","OSError: [Errno 107] Transport endpoint is not connected\n","\n","During handling of the above exception, another exception occurred:\n","\n","Traceback (most recent call last):\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/interactiveshell.py\", line 1823, in showtraceback\n","    stb = value._render_traceback_()\n","AttributeError: 'OSError' object has no attribute '_render_traceback_'\n","\n","During handling of the above exception, another exception occurred:\n","\n","Traceback (most recent call last):\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/ultratb.py\", line 1132, in get_records\n","    return _fixed_getinnerframes(etb, number_of_lines_of_context, tb_offset)\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/ultratb.py\", line 313, in wrapped\n","    return f(*args, **kwargs)\n","  File \"/usr/local/lib/python3.6/dist-packages/IPython/core/ultratb.py\", line 358, in _fixed_getinnerframes\n","    records = fix_frame_records_filenames(inspect.getinnerframes(etb, context))\n","  File \"/usr/lib/python3.6/inspect.py\", line 1490, in getinnerframes\n","    frameinfo = (tb.tb_frame,) + getframeinfo(tb, context)\n","  File \"/usr/lib/python3.6/inspect.py\", line 1448, in getframeinfo\n","    filename = getsourcefile(frame) or getfile(frame)\n","  File \"/usr/lib/python3.6/inspect.py\", line 696, in getsourcefile\n","    if getattr(getmodule(object, filename), '__loader__', None) is not None:\n","  File \"/usr/lib/python3.6/inspect.py\", line 725, in getmodule\n","    file = getabsfile(object, _filename)\n","  File \"/usr/lib/python3.6/inspect.py\", line 709, in getabsfile\n","    return os.path.normcase(os.path.abspath(_filename))\n","  File \"/usr/lib/python3.6/posixpath.py\", line 383, in abspath\n","    cwd = os.getcwd()\n","OSError: [Errno 107] Transport endpoint is not connected\n"],"name":"stdout"},{"output_type":"error","ename":"OSError","evalue":"ignored","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m"]}]},{"cell_type":"code","metadata":{"id":"rLEHqcebsci3","colab_type":"code","colab":{}},"source":[""],"execution_count":null,"outputs":[]}]}